test/test_iterdatapipe.py

```diff
@@ -951,6 +952,30 @@ def test_mux_longest_iterdatapipe(self):
         with self.assertRaises(TypeError):
             len(output_dp)
 
+    def test_extractor(self):
```

Comment:
```suggestion
    def test_key_extractor(self):
```
nit: We used to have a different `extractor`